### PR TITLE
Automatically set CKEDITOR_BASEPATH

### PR DESCRIPTION
**Pull request by matthiask**
This would require reversing the order of `ckeditor-init.js` and `ckeditor.js`.

### CODE REVIEW

File: ckeditor/static/ckeditor/ckeditor-init.js [hunk 1]
```diff
@@ -7,6 +7,12 @@
   }
 
   var $ = jQuery || djangoJQuery;
+
+  var el = $('.django-ckeditor-widget');
```
Comment (Flimm) [inline ckeditor/static/ckeditor/ckeditor-init.js]: At this point, the div with the class `django-ckeditor-widget` has not yet been loaded, so `el.length` will be zero. Perhaps this code should be moved into the jQuery ready function.

**Comment by Flimm** [comment]
This doesn't work for me. I've added a comment to the particular line of code where the problem is.

**Comment by riklaunim** [comment]
Then we shall venture forth into the JavaScript nightmare and vanquish the source of this evil.

**Comment by matthiask** [comment]
My bad, it obviously didn't test this well enough. I'll have to determine
why it worked for me first...

**Pull request merged**